gravityflow/gravityflowincomingwebhook
# Add support for creating entries

This PR adds support for creating entries via webhooks. This differs from the Gravity Forms `POST forms/[form ID]/submissions` endpoint in that it doesn't pass through the submission process which fires notifications and feeds, it just creates an entry and kicks off the workflow like the New Entry step.

**Testing instructions**
Add a webhook from the Incoming Webhook form settings tab. Send a `POST` request via Postman or similar to create an entry via the webhook. Ensure that the fields are mapped correctly and the workflow starts as expected. Check for regressions.
